string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,34 @@
+/* exported titleCase */
+function titleCase(title) {
+  let arr = title.toLowerCase();
+  arr = arr.split(' ');
+  for (var i = 0; i < arr.length; i++) {
+    if (arr[i] === 'the' && i === 0) {
+      arr[i] = arr[i][0].toUpperCase() + arr[i].slice(1);
+    } else if (i !== 0 && arr[i - 1].includes(':')) {
+      arr[i] = arr[i][0].toUpperCase() + arr[i].slice(1);
+    } else if (arr[i] === 'the' && i !== 0) {
+      arr[i].toLowerCase();
+    } else if (arr[i] === 'for') {
+      arr[i].toLowerCase();
+    } else if (arr[i] === 'self-taught') {
```

Comment:
🙅 . `'self-taught'` isn't one of the known words. You shouldn't hard-code it.

For `titleCase`, I recommend changing your strategy a bit. Make an array of the known short words. Make another array for the dictionary words given to you (api, javascript).

An array has an `includes()` method. So you can use that to see if the word you're checking is a short word or a dictionary word.